src/python/grpcio/grpc/__init__.py

```diff
@@ -1252,6 +1269,65 @@ def ssl_server_credentials(private_key_certificate_chain_pairs,
             ], require_client_auth))
 
 
+def ssl_server_credentials_with_cert_config_fetcher(cert_config_fetcher_cb,
+                                                    require_client_auth=False):
+    """Creates a ServerCredentials for use with an SSL-enabled Server.
+
+    Args:
+      cert_config_fetcher_cb (callable): a callback that takes no
+        arguments and should return a ServerCertificateConfig to
+        replace the server's current cert, or None for no change
+        (i.e., the server will continue its current certificate
+        config). The library will call this callback once during
+        server instantiation to obtain the initial certificate
+        config---a valid ServerCertificateConfig must be returned in
+        this case. Then, the library will call this callback on
+        *every* new client connection before starting the TLS
+        handshake with the client, thus allowing the user application
+        to optionally return a new ServerCertificateConfig that the
+        server will then use for the handshake.
+      require_client_auth: A boolean indicating whether or not to
+        require clients to be authenticated. May only be True if
+        root_certificates is not None.
+
+    Returns:
+      A ServerCredentials for use with an SSL-enabled
+      Server. Typically, this object is an argument to
+      add_secure_port() method during server setup.
+    """
+
+    if not callable(cert_config_fetcher_cb):
+        raise ValueError('cert_config_fetcher_cb must be callable')
+    return ServerCredentials(
+        _cygrpc.server_credentials_ssl_with_cert_config_fetcher(
+            cert_config_fetcher_cb, require_client_auth))
+
+
+def ssl_server_certificate_config(private_key_certificate_chain_pairs,
+                                  root_certificates=None):
+    """Creates a ServerCertificateConfig for use with an SSL-enabled Server.
+
+    Args:
+      private_key_certificate_chain_pairs: A list of pairs of the form
```

Comment:
Why a list rather than a sequence, collection, or iterable?

Reply:
just copy-paste from `def ssl_server_credentials`. please let me know what you like to change the doc-string to.